pkg/minishift/registration/redhat.go

```diff
@@ -87,15 +86,16 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 				param.Username,
 				minishiftStrings.EscapeSingleQuote(param.Password))
 
-			// start timed SSH command to register
-			spinnerView.Start()
+			cmdUtil.ShowProgressDots(progressDots)
 			startTime := time.Now()
+			// start timed SSH command to register
 			_, err = registrator.SSHCommand(subscriptionCommand)
-			spinnerView.Stop()
+			// stops the progress of dots
+			progressDots <- true
 			if err == nil {
-				fmt.Print("Registration successful ")
+				fmt.Print(" OK ")
 			} else {
-				fmt.Print("Registration unsuccessful ")
+				fmt.Print(" FAIL ")
 			}
 			fmt.Println(util.TimeElapsed(startTime, true))
```

Comment:
If you want brackets, you would have to put them here...

The function https://github.com/budhrg/minishift/blob/03606c244dcf5030a2e0c2d357d7f0f888fb2c59/pkg/util/utils.go#L133 is a util function, and would need to produce a result that can be processed easily without side-effects or markup.

Reply:
Sure